nixos/modules/system/boot/networkd.nix

```diff
@@ -1073,7 +1073,7 @@ in
 
       systemd.services.systemd-networkd = {
         wantedBy = [ "multi-user.target" ];
-        restartTriggers = attrNames unitFiles;
+        restartTriggers = map (x: x.source) (attrValues unitFiles);
```

Comment:
Please add a more descriptive commit message here. Proposal:

```
nixos/networkd: Fix restartTriggers

1d61efb7f177f7b70c467ab4940fde0a3481d4dc accidentially changed the
restartTriggers of systemd-networkd.service` to point to the attribute name (in
this case, a location relative to `/etc`), instead of the location of the
.network file in the nix store.

This caused systemd-networkd to not get restarted on activation of new
networking config, if the file name hasn't changed.

Fix this, by pointing this back to the location in the nix store.
```